src/js/angular/core/directives/copy-to-clipboard/copy-to-clipboard.directive.js

```diff
@@ -41,7 +41,25 @@ copyToClipboard.$inject = ['$translate', 'toastr'];
  */
 function copyToClipboard($translate, toastr) {
     return {
-        template: '<button class="copy-btn" gdb-tooltip="{{tooltipText | translate}}" ng-click="copyToClipboard()"><i class="fa-regular fa-clone"></i></button>',
+        template: `
+            <style>
+                .copy-btn {
+                    border: none;
+                    background-color: transparent;
+                    cursor: pointer;
+                    padding: 0;
+                    color: var(--secondary-color);
+                }
+                .copy-btn:hover {
+                    transform: scale(1.1);
+                    transition: all 0.1s ease-out;
+                }
+                .copy-btn:focus {
+                    outline: none;
+                }
+            </style>
+            <button class="copy-btn" gdb-tooltip="{{tooltipText | translate}}" ng-click="copyToClipboard()"><i class="fa-regular fa-clone"></i></button>
```

Comment:
We have similar styling applied in 'agent-settings-modal.css' and 'chat-item-details'. We could clean them up.

Reply:
they are different, maybe the color and others - I haven't compared them